pandas/tests/io/test_parquet.py

```diff
@@ -473,6 +473,18 @@ def test_partition_cols_supported(self, pa, df_full):
             assert len(dataset.partitions.partition_names) == 2
             assert dataset.partitions.partition_names == set(partition_cols)
 
+    def test_partition_cols_string(self, pa, df_full):
+        # GH #23283
+        partition_cols = "bool"
+        df = df_full
+        with tm.ensure_clean_dir() as path:
+            df.to_parquet(path, partition_cols=partition_cols, compression=None)
+            import pyarrow.parquet as pq
+
+            dataset = pq.ParquetDataset(path, validate_schema=False)
+            assert len(dataset.partitions.partition_names) == 1
+            assert dataset.partitions.partition_names == set([partition_cols])
```

Comment:
I'm surprised running `black` locally didn't catch this. 

Basically, PEP8 is crazy annoying: https://pypi.org/project/flake8-comprehensions/ - see "C405-406,C409-410: Unnecessary list/tuple literal"
 
You'll have to change line 486 to be:
```
assert dataset.partitions.partition_names == set((partition_cols))
```

Reply:
Fixed with latest commit